packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -226,12 +227,34 @@ export async function batchInstallFontFaces( fontFamilyId, fontFacesData ) {
 	return results;
 }
 
+/*
+ * Makes a font face filename based on the font face properties.
+ */
+function makeFileNameFromFontFace( fontFace ) {
+	const properties = [
+		'fontFamily',
+		'fontStyle',
+		'fontWeight',
+		'unicodeRange',
```

Comment:
Note that `fontStretch` is also needed here to be more in line with the css spec for font matching.

Reply:
Assuming these properties are concatenated to create the final Google Fonts file name **_only_**. Then I would suggest we leave out `fontStretch`.  I don't think we need every property in the file name and it'd get looooonnnnnggg.

Reply:
We could slightly reduce the length of these filenames by using aliases for the font styles, something like:

- italic = `ital` or `it`
- normal = `norm` or `nor`
- oblique = `obli` or `obl`

It's only a tiny change but it may make a difference on particularly long file names.

We could also consider using pascal-case on the font family name (e.g. `AROneSans` rather than `ar-one-sans`) to further reduce the filenames. The properties could still be separated by dashes, like `AdLaMDisplay-normal-400.woff2` instead of `ad-la-m-display-normal-400.woff2`.